src/uct/ib/dc/dc_mlx5.c

```diff
@@ -972,11 +972,12 @@ ucs_status_t uct_dc_mlx5_iface_fc_handler(uct_rc_iface_t *rc_iface, unsigned qp_
 
         status = uct_dc_mlx5_iface_fc_grant(&dc_req->super.super);
         if (status == UCS_ERR_NO_RESOURCE){
-            status = uct_ep_pending_add(&ep->super.super, &dc_req->super.super,
-                                        0);
+            uct_dc_mlx5_ep_pending_common(iface, ep, &dc_req->super.super, 0, 1);
```

Comment:
👍 for making common function
note that order of grants may be reversed now, but i guess it doesn't matter because there is no information in the grant message itself